packages/app/src/app/pages/Pro/index-v2.tsx

```diff
@@ -23,14 +23,16 @@ export const ProPage: React.FC = () => (
   <ThemeProvider>
     <Helmet>
       <title>Pro - CodeSandbox</title>
+      <script src="https://cdn.paddle.com/paddle/paddle.js" />
```

Comment:
script tag, i know :(

Reply:
Then do something like this: https://github.com/codesandbox/codesandbox-client/blob/homepage-pro/packages/app/src/app/components/SubscribeForm/index.tsx#L31

Reply:
love it!